allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_transforms.py

```diff
@@ -262,13 +292,50 @@ def get_extended_trials(self) -> pd.DataFrame:
 
     @memoize
     def get_experiment_date(self) -> datetime:
-        """Return timestamp the behavior stimulus file began recording in UTC
+        """Return the timestamp for when experiment was started in UTC
+
+        NOTE: This method will only get acquisition datetime from
+        extractor (data from LIMS) methods. As a sanity check,
+        it will also read the acquisition datetime from the behavior stimulus
+        (*.pkl) file and raise a warning if the date differs too much from the
+        datetime obtained from the behavior stimulus (*.pkl) file.
+
         :rtype: datetime
         """
-        data = self._behavior_stimulus_file()
-        # Assuming file has local time of computer (Seattle)
-        tz = pytz.timezone("America/Los_Angeles")
-        return tz.localize(data["start_time"]).astimezone(pytz.utc)
+        extractor_acq_date = self.extractor.get_experiment_date()
+
+        pkl_data = self._behavior_stimulus_file()
+        pkl_raw_acq_date = pkl_data["start_time"]
+        if isinstance(pkl_raw_acq_date, datetime):
+            pkl_acq_date = pytz.utc.localize(pkl_raw_acq_date)
+
+        elif isinstance(pkl_raw_acq_date, (int, float)):
+            # We are dealing with an older pkl file where the acq time is
+            # stored as a Unix style timestamp string
+            parsed_pkl_acq_date = datetime.fromtimestamp(pkl_raw_acq_date)
+            pkl_acq_date = pytz.utc.localize(parsed_pkl_acq_date)
+        else:
+            pkl_acq_date = None
+            warnings.warn(
+                "Could not parse the acquisition datetime "
+                f"({pkl_raw_acq_date}) found in the following stimulus *.pkl: "
+                f"{self.extractor.get_behavior_stimulus_file()}"
+            )
+
+        if pkl_acq_date:
+            acq_start_diff = (
+                extractor_acq_date - pkl_acq_date).total_seconds()
+            # If acquisition dates differ by more than an hour
+            if abs(acq_start_diff) > 360:
+                warnings.warn(
+                    "The `date_of_acquisition` field in LIMS "
+                    f"({extractor_acq_date}) for behavior session "
+                    f"({self.get_behavior_session_id()}) deviates by more than"
```

Comment:
Slight typo. Extra space needed at the end

Reply:
Good catch, will update.